src/ui/libs/DatalensChartkit/components/Control/Control.tsx

```diff
@@ -152,6 +153,17 @@ class Control<TProviderData> extends React.PureComponent<
         );
     }
 
+    async runAction(args: StringParams) {
+        const {runAction, onAction} = this.props;
```

Comment:
Could you rename one of `runAction`'s to get rid of variables shadowing? Or you can use inner function in this way `this.props.runAction`